sql/testdata/builtin_function

```diff
@@ -84,3 +84,107 @@ SELECT SUBSTR('12345', -2, 77)
 
 statement error SUBSTR: negative substring length -1 not allowed
 SELECT SUBSTR('12345', 2, -1)
+
+query T
+SELECT concat_ws(',', 'abcde', 2, NULL, 22)
+----
+abcde,2,22
+
+query T
+SELECT split_part('abc~@~def~@~ghi', '~@~', 2)
+----
+def
+
+query T
+SELECT repeat('Pg', 4)
+----
+PgPgPgPg
+
+query T
+SELECT repeat('Pg', -1) || 'empty'
+----
+empty
+
+query T
+SELECT ascii('x')
+----
+120
+
+query T
+SELECT md5('abc')
+----
+900150983cd24fb0d6963f7d28e17f72
+
+query T
+SELECT to_hex(2147483647)
+----
+7fffffff
+
+query T
+SELECT strpos('high', 'ig')
+----
+2
+
+query T
+SELECT btrim('xyxtrimyyx', 'xy')
+----
+trim
+
+query T
+SELECT btrim('   postgres  ')
+----
+postgres
+
+query T
+SELECT ltrim('zzzytrimxyz', 'xyz')
+----
+trimxyz
+
+query T
+SELECT ltrim('   trimxyz')
+----
+trimxyz
+
+query T
+SELECT rtrim('zzzytrimxyz', 'xyz')
+----
+zzzytrim
+
+query T
+SELECT rtrim('zzzytrimxyz   ')
+----
+zzzytrimxyz
+
+query T
+SELECT reverse('abcde')
+----
+edcba
+
+# TODO(XisiHuang): replace is a keyword, causes this test fail
```

Comment:
I think our lookup of function names is incorrect. We use `expr.Name.String()` where `Name` is a `QualifiedName`, but that will quote all keywords. See #2108 about the unnecessary quoting. But regardless of the quoting, the `QualifiedName` should be verified to not contain an indirection and then we can use `string(expr.Name.Base)` instead.